sdk/monitor/opentelemetry-exporter/test/common/assert.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as assert from "assert";
```

Comment:
nit: we prefer Chai's assert function since it handles more cases + has better logs than the node equivalent.

I realize this could be hard to migrate to, but I thought I'd mention it. :)